### PR TITLE
Feature/daniel contreras form exercise

### PR DESCRIPTION
**Pull request by danielContrerasL**
This is a form to get user information for aply to subsidy

### CODE REVIEW

**Review by pushdev-code — CHANGES_REQUESTED** [review]
I see your form exercise is completed, but please make sure you have one exercise Per PR, I see the basics and the semantics exercises in this one, so please don't include them in these set of changes.

File: 4-forms-validations/form.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        <meta charset="utf-8">
+        <title>Government Subsidy</title>
+    </head>
+<body>
+    <header>
+        <h1>Government Subsidy</h1>
+        <h2>Subcide for athletes</h2>
+    </header>
+    <main>
+        <p>This is a form to apply for a sport subsidy</p>
```
Comment (pushdev-code) [inline 4-forms-validations/form.html]: this should be a `<h1>`

File: 4-forms-validations/form.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                    <input required type="email" id="email" name="user_email">
+                </label>
+            </fieldset>
+            <label for="birthDate">
+                Select your birth date:
+                <input required type="date" id="birthDate" name="user_birth_date" value="2020-04-24" min="1920-01-01" max="2020-12-31">
+            </label>
+            <fieldset>
+                <legend>Choose your gender: </legend>
+                <select id="gender" required>
```
Comment (pushdev-code) [inline 4-forms-validations/form.html]:
```suggestion
                <label for="gender">Gender: </label><select id="gender" required>
```

File: 4-forms-validations/form.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                    Swimming
+                </label>
+                <label for="volleyball"><input type="checkbox" id="volleyball" value="volleyball">
+                    volleyball
+                </label>
+            </fieldset>
+            <label for="profile_picture">Choose a profile picture:</label>
+            <input required type="file" id="profile_picture" name="profile_picture" accept="image/png, image/jpeg">
+            <br>
+            <button type="submit" name="send" id="btm-send">Send</button>
```
Comment (pushdev-code) [inline 4-forms-validations/form.html]:
```suggestion
            <button name="send" id="btm-send">Send</button>
```

**Review by pushdev-code — CHANGES_REQUESTED** [review]
Some small comments left, please move accessible exercise to its own PR.

**Comment by juancho11gm** [comment]
The PR still containing the `3-semantic-html/basic.html` and the `5-Accessibility/accessible-form.html`.

**Review by pushdev-code — APPROVED** [review]

**Pull request merged**